Content.Server/Shipyard/Systems/ShipyardSystem.Consoles.cs

```diff
@@ -173,7 +173,7 @@ private void OnPurchaseMessage(EntityUid uid, ShipyardConsoleComponent component
             var newAccess = newCap.Tags.ToList();
             newAccess.Add($"Captain");
 
-            if (ShipyardConsoleUiKey.Security == (ShipyardConsoleUiKey) args.UiKey)
+            if (ShipyardConsoleUiKey.Security == (ShipyardConsoleUiKey) args.UiKey || ShipyardConsoleUiKey.CustomSecurity == (ShipyardConsoleUiKey) args.UiKey)
```

Comment:
This is bad. It was a bad idea to hardcode this property of secyard and blackmarket consoles in the first place. Instead you should add a new property to ShipyardConsoleComponent specifying the radio channel, or create a new component to specify it.

Reply:
Yep it was a temp code work, even at the time I said it need actual rework

Reply:
Right, I can remove the change from this PR entirely since again this seems to need a seperate PR, but Ill look into a softcoded solution then.